fw_gear_hcp_diff/args/DiffPreprocPipeline.py

```diff
@@ -0,0 +1,383 @@
+"""
+Builds, validates, and excecutes parameters for the HCP script
+/opt/HCP-Pipelines/DiffusionPreprocessing/DiffPreprocPipeline.sh
+"""
+
+import logging
+import os
+import os.path as op
+from collections import OrderedDict
+
+from tr import tr
+
+from utils.diff_utils import make_sym_link
+from ..gear_preliminaries import create_sanitized_filepath
+from .common import build_command_list, exec_command
+
+log = logging.getLogger(__name__)
+
+
+def build(context):
```

Comment:
I would encourage typing here, just for readability.  I think this is a gear context, but it would be helpful to have either 
* `def build(context: flywheel_gear_toolkit.GearToolkitContext) -> None`
* A docstring such as:
```python
def build(context):
"""Builds context for script.

Args:
    context (GearToolkitContext)

Returns
    (None)
"""
```